Server/pom.xml

```diff
@@ -298,5 +298,11 @@
 		    <groupId>javax.annotation</groupId>
 		    <artifactId>javax.annotation-api</artifactId>
 		</dependency>
+        <!-- https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc -->
+        <dependency>
+            <groupId>com.microsoft.sqlserver</groupId>
+            <artifactId>mssql-jdbc</artifactId>
+            <version>12.6.2.jre11</version>
+        </dependency>
```

Comment:
Please remove this import as it loads a library into the installler package that most people will never use and is not part of the default OpenAS2 functionality.
Anyone wishing to use packages other than the standard packages provided by OpenAS2 should create their own wrapper installer otherwise the OpenAS2 install package will become bloated unnecessarily.